app/src/main/java/org/oppia/app/onboarding/OnboardingActivityPresenter.kt

```diff
@@ -0,0 +1,30 @@
+package org.oppia.app.onboarding
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import kotlinx.android.synthetic.main.dummy_onboard.*
```

Comment:
Avoid using `synthetic` imports

Reply:
updated to import kotlinx.android.synthetic.main.dummy_onboard.onBoard